jitpack.yml

```diff
@@ -0,0 +1,4 @@
+jdk:
```

Comment:
Ideally the tests would not depend on Jitpack. Unfortunately, the service bean has to be compiled beforehand and provided as an external bean archive JAR, so it has to be served from a Maven repository.